app/src/common/hooks/useMergeConceptMapsToMappings.ts

```diff
@@ -0,0 +1,102 @@
+import { useApiConceptMapsListQuery } from "services/api/endpoints";
+import type { Mapping } from "services/api/generated/api.generated";
+
+const useMergeConceptMapsToMappings = (params: {
```

Comment:
We still don't have tests for our apps, right? 😅

Reply:
I will add a test!